runtime/doc/builtin.txt

```diff
@@ -4306,8 +4305,9 @@ getregion({pos1}, {pos2} [, {opts}])			*getregion()*
 		- If the region is blockwise and it starts or ends in the
 		  middle of a multi-cell character, it is not included but
 		  its selected part is substituted with spaces.
-		- If {pos1} or {pos2} is not current in the buffer, an empty
+		- If {pos1} and {pos2} are not in the same buffer, an empty
 		  list is returned.
+		- {pos1} and {pos2} must be exists in tabs.
```

Comment:
`{pos1} and {pos2} must be loaded in a window within any tab?

Reply:
Fixed.